assets/src/block-editor/components/sidebar-notification/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.css';
+import { Loading } from '../../../components/loading';
+
+/**
+ * Notification component used in the block editor sidebar.
+ *
+ * @param {Object} props
+ * @param {string|Object} props.action Call to action element.
+ * @param {string|Object} props.icon Status icon element.
```

Comment:
Are strings ever really passed here?

```suggestion
 * @param {Object} props.action Call to action element.
 * @param {Object} props.icon Status icon element.
```